src/Utils/Hooks/useCurrentTime.tsx

```diff
@@ -20,11 +20,13 @@ import { useEffect, useRef, useState } from "react"
 interface UseCurrentTimeProps {
   interval?: number
   syncWithServer?: boolean
+  enabled?: boolean
 }
 
 export const useCurrentTime = ({
-  interval = 1000,
+  interval = 5000,
```

Comment:
This is used to display ![](https://capture.static.damonzucconi.com/Screen-Shot-2025-01-08-09-43-50.29-zFgjEKtZriXZXmNzRMDUvGvOC9GI6gn9X8dJcumf5zmVvU1ca35Zv2jARoANiZ5CQRQKFYiOEj0Ir2LYniYGfknnUQGNM80Q55f7.png) so at the very least, that usage of it needs the 1 second interval